api/available-routes.js

```diff
@@ -0,0 +1,117 @@
+const { getLogger, InputError } = require("./_utils");
+const enabledRoutesAsJson = require("../src/data/routes_1_0xc186fA914353c44b2E33eBE05f21846F1048bEda.json");
+
+const handler = async (request, response) => {
+  const logger = getLogger();
+  try {
+    let { originChainId, destinationChainId, originToken, destinationToken } =
```

Comment:
I think this can be const too

```suggestion
    const { originChainId, destinationChainId, originToken, destinationToken } =
```